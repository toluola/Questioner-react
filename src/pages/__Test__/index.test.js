import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import MainPage from '../index';

const mockStore = configureMockStore([thunk]);
const store = mockStore(jest.fn());

const props = {
    isAuthenticated: jest.fn()   
   }

describe('Should render index component', () => {
    test('<Index /> component is rendered', () => {
const IndexComponent = shallow(<Provider store={store}><BrowserRouter><MainPage {...props}/></BrowserRouter></Provider>);
        expect(IndexComponent).toBeTruthy();
        expect(IndexComponent).toMatchSnapshot();
    });
});