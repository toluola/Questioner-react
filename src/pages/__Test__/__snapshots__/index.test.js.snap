// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render index component <Index /> component is rendered 1`] = `ShallowWrapper {}`;
